docs/field-values.asciidoc

```diff
@@ -298,6 +299,18 @@ Use this category of events to visualize and analyze process-specific informatio
 access, change, end, info, start
 
 
+[float]
+[[ecs-event-category-registry]]
+==== registry
+
+Having to do with settings and assets stored in the Windows registry. Use this category to visualize and analyze activity such as registry access and modifications.
```

Comment:
@MikePaquette Not word for word our description from way back when, but here's what I'd be going for now.

Reply:
Comparing this description to the draft in the public ECS categorization, I think limiting the category's intent to solely the Windows registry makes good sense.

Reply:
I don't think there's anything else that has a registry, is there? If I'm wrong on that, then perhaps we should expand that again.

Reply:
there are package registries. I can see how an individual package could make sense in ECS, but not as much for a package registry

Reply:
Container registries were another one. But no, I don't think we need to expand the category's intent.

Reply:
Yes my question wasn't so much about usage of the word registry in other contexts. More around are there other OSes that have a weird settings database/object storage thingy called a registry.

But yeah, being reminded about these other current day uses of the word registry reinforces my resolve on the wording changes :-) So let's go with that